### PR TITLE
Configure vnc as default NOVA console

### PR DESCRIPTION
**Pull request by BjoernT**
This PR will enable the embedded noVNC console over the SPICE
as the nova default console.
With the override nova_console_type the nova default console type
can be switched to VNC (novnc) or SPICE (spice).

Connects #771

### CODE REVIEW

File: rpcd/etc/openstack_deploy/user_variables.yml [hunk 1]
```diff
@@ -85,3 +85,6 @@ nova_ceilometer_enabled: False
 swift_ceilometer_enabled: False
 keystone_ceilometer_enabled: False
 tempest_service_available_ceilometer: False
+
+#Enable noVNC over spice console
```
Comment (prometheanfire) [inline rpcd/etc/openstack_deploy/user_variables.yml]: # Enable noVNC instead of Spice

space after the # (formatting thing) and clearer wording

Comment (BjoernT) [inline rpcd/etc/openstack_deploy/user_variables.yml]: yeah will fix that with the next update

**Comment by BjoernT** [comment]
recheck

**Comment by d34dh0r53** [comment]
recheck

**Comment by d34dh0r53** [comment]
I think the F5 upgrade script should be in a different PR.

**Comment by BjoernT** [comment]
Yeah we would need the f5-upgrade for liberty only so I can PR that for that branch only. At least the F5 upgrade script.
The F5 config has to updated with this PR

**Comment by prometheanfire** [comment]
recheck - network errors

**Comment by d34dh0r53** [comment]
recheck

**Comment by BjoernT** [comment]
recheck

**Comment by prometheanfire** [comment]
recheck_ceph
recheck_swift

**Comment by BjoernT** [comment]
recheck_ceph
recheck_swift

**Comment by git-harry** [comment]
I'm closing this because it relates to an issue that has been closed and does not currently work nor does if form part of the next two releases. I have referenced it on the new issue that is tracking this work so that the work done in this pull request is not lost.

**Comment by BjoernT** [comment]
So please leave it open we will use the exact commit for a 13.x release

**Comment by git-harry** [comment]
Marking it as closed is for the purpose of maintaining a tidy repo and not a comment on the pull request itself. Being closed is not the same as being deleted. This pull request cannot be used as is because it does not merge and even if that was addressed now it would likely need doing again in a few months time when novnc will form part of r13.2.0. Also, as said, I have referenced this pull request on the issue that is being used to track the novnc work item and so it will be available to those assigned to complete it.

**Pull request closed without merging**